.env

```diff
@@ -0,0 +1,12 @@
+#database
+DB_HOST=db_server
+DB_PORT=5432
+DB_NAME=calendar
+DB_USER=postgres
+DB_PASS=1234
+#app
+APP_PORT=8033
+#pg_admin
+PGADMIN_DEFAULT_EMAIL=pgadmin4@pgadmin.org
+PGADMIN_DEFAULT_PASSWORD=admin
+PGADMIN_PORT=5050
```

Comment:
Add this file to .gitignore
.env files should not be included in repositories. 
Exceptions are: .env.example files that include only keys without values